src/services/errors.ts

```diff
@@ -0,0 +1,37 @@
+interface Errors {
```

Comment:
I want you to use the opportunity to learn object oriented programming (OOP).

Create a folder called `Expecptions`. In that folder, you will create one file for each kind of custom Exception you wish to throw in the application. Each file will contain the class for that exception. Here are standard exception class I expect.

- ClientErrorException (400)
- NotFoundException (404)
- InvalidBodyParameterException (422)
- ConflictException (409)
- UnauthorizedException (403)
- UnauthenticatedException (401)
- ServerError (500)

So take the time, learn OOP and then come back and implement this using it.